scipy/integrate/_ivp/lsoda.py

```diff
@@ -168,10 +168,12 @@ def _dense_output_impl(self):
         iwork = self._lsoda_solver._integrator.iwork
         rwork = self._lsoda_solver._integrator.rwork
 
-        order = iwork[14]
+        order = iwork[13]
```

Comment:
Is this necessary for the original issue fixed by this PR?

Reply:
Yes, I believe this is required. From reading what I wrote previously, it sounds to me that it allows to pick the right number of terms in the Taylor expansion.

Reply:
I think I misunderstood because of the force push; I thought it was new.